backend/src/app/rpc/commands/binfile.clj

```diff
@@ -511,14 +523,19 @@
 
   (doseq [file-id (-> *state* deref :files)]
     (let [detach? (and (not embed-assets?) (not include-libraries?))
-          file    (cond-> (get-file cfg file-id)
-                    detach?
-                    (-> (ctf/detach-external-references file-id)
-                        (dissoc :libraries))
-                    embed-assets?
-                    (update :data embed-file-assets cfg file-id))
+          thumbnails (get-file-tagged-object-thumbnails cfg file-id)
+          file  (cond-> (get-file cfg file-id)
+                  detach?
+                  (-> (ctf/detach-external-references file-id)
+                      (dissoc :libraries))
+
+                  embed-assets?
+                  (update :data embed-file-assets cfg file-id)
 
-          media   (get-file-media cfg file)]
+                  true
```

Comment:
The convention is use `:always` keyword for indicate that this branch should be executed always